BMI_DAO.kt

```diff
@@ -0,0 +1,14 @@
+interface BMI_DAO {
+    val _height: Double
```

Comment:
Starting variables with _ indicates that they are private, but the private modifier is missing, so these default to public.  Add the private modifier.